tsconfigdoc.json

```diff
@@ -233,6 +233,9 @@
     {
       "path": "./packages/statusbar-extension"
     },
+    {
+      "path": "./packages/tabmanager-extension"
+    },
```

Comment:
Please remove this bit, it is causing test failures. `tabmanager-extension` was removed in 3.0.

```suggestion
```